builds/jazz-build-module/utility-loader.groovy

```diff
@@ -256,7 +262,18 @@ def createAllStackResources (whiteListModule, events, config, stackResources, en
 			}
 		}
 	}
-	return lambdaArns
+	echo "arnsMap: $arnsMap"
+	return arnsMap
+}
+
+def archiveCustomRole(assets_api, auth_token, config, env, events) {  
+  def assets = getAssets(assets_api, auth_token, config, env)
+  def assetList = parseJson(assets)
+
+  for (asset in assetList.data.assets) {
+	if (asset.asset_type == 'iam_role')
+      	events.sendCompletedEvent('UPDATE_ASSET', "User specific role is using.", generateAssetMap(asset.provider, asset.provider_id, "iam_role", config), env)
```

Comment:
Does the completedEvent archive the asset?
`"User specific role is using."` ==> what does this message mean?

Reply:
Yes it will archive the asset.  I had changed the comment